### PR TITLE
Update Turkish translation

### PR DESCRIPTION
**Pull request by mahmutelmas06**
Updated to latest minetest.po pulled from po/jp because the first release i made was based on minetest.pot which is outdated and had missing strings.

### CODE REVIEW

**Comment by ShadowNinja** [comment]
```
po/tr/minetest.po:481: `msgid' and `msgstr' entries do not both end with '\n'
/usr/bin/msgfmt: found 1 fatal error
```

File: po/tr/minetest.po [hunk 1]
```diff
@@ -12,1237 +17,832 @@ msgstr ""
...
+"- Space: jump/climb\n"
+"- Shift: sneak/go down\n"
+"- Q: drop item\n"
+"- I: inventory\n"
+"- Mouse: turn/look\n"
+"- Mouse left: dig/punch\n"
+"- Mouse right: place/use\n"
+"- Mouse wheel: select item\n"
+"- T: chat\n"
+msgstr "Menü"
```
Comment (Rui914) [inline po/tr/minetest.po]: You need to write a description of the operation here.

Comment (mahmutelmas06) [inline po/tr/minetest.po]: updated

**Comment by mahmutelmas06** [comment]
Thnks i have updated the file.
I am using EazyPo editor and it didnt give me a compile error so i missed.



Mahmut Elmas

Date: Sun, 1 Mar 2015 14:41:27 -0800

**Comment by Rui914** [comment]
Build success!
Thank you to fix.

**Comment by ShadowNinja** [comment]
You've removed a lot of comment lines that are wanted.

**Comment by mahmutelmas06** [comment]
Yeah i figured that out and fixed, updated and Rui914 merged. Thank you for interest

**Comment by Rui914** [comment]
Why was closed?
This commit has not yet been merged.

**Comment by mahmutelmas06** [comment]
Ouch sorry i think he already merged :disappointed:

**Comment by ShadowNinja** [comment]
Use `util/updatepo.sh` to add all of the latest translations, and then edit the relevant parts in the po file.

**Comment by nerzhul** [comment]
please squash your commits

**Comment by mahmutelmas06** [comment]
I have squashed but didnt understand what is that compile error

**Pull request closed without merging**